monitoring/OpenShift.md

```diff
@@ -133,7 +133,9 @@ Specify `OPENSHIFT_PATH_ROUTES=true` in the `$USER_DIR/user.env` file
 file (for only logging components) to use path-based routes.
 
 The Grafana hostname can be configured using `OPENSHIFT_ROUTE_HOST_GRAFANA` in
-`$USER_DIR/user.env` or `$USER_DIR/monitoring/user.env`.
+`$USER_DIR/user.env` or `$USER_DIR/monitoring/user.env`. Note that OpenShift
+does not allow the use of the same across namespaces, so do not use the same
```

Comment:
see comment above.

Reply:
Ugh...sorry. Not sure how that happened.

Fixed to: "Note that OpenShift does not allow the use of the same route hostname across namespaces, so do not use the same custom hostname across logging and monitoring."